index.py

```diff
@@ -8,7 +8,8 @@
 
 
 
-app.config['APPLICATION_ROOT'] = os.environ.get('ROOT_PREFIX', '/applist')
+# app.config['APPLICATION_ROOT'] = os.environ.get('ROOT_PREFIX', '/applist')
+app.config['APPLICATION_ROOT'] = '/applist'
```

Comment:
This is now hardcoded?  It might break the current CI deploy.

Reply:
...I can change CI to use /applist but it might be nicer to have it customizable.

Reply:
ok I will put it back

Reply:
...actually I think we changed it in CI already.